project/src/components/user-comment/user-comment.tsx

```diff
@@ -0,0 +1,52 @@
+import { CommentType } from '../../types/comment';
+import { computeRatingWidth } from '../../utils';
+import { formatDateYYYYMMDD, formatDateMMMMYYYY } from '../../utils';
+
+type AppScreenProps = {
+  comment: CommentType;
+}
+
+function UserComment({comment}: AppScreenProps): JSX.Element {
+  const {
+    comment: commentText,
+    date,
+    rating,
+    user} = comment;
+  const {avatarUrl, isPro, name} = user;
+
+  return (
+    <li className="reviews__item">
+      <div className="reviews__user user">
+        <div className={`reviews__avatar-wrapper ${isPro && 'reviews__avatar-wrapper--pro'} user__avatar-wrapper`}>
+          <img
+            className="reviews__avatar user__avatar"
+            src={avatarUrl}
+            width="54"
+            height="54"
+            alt="Reviews avatar"
+          />
+        </div>
+        <span className="reviews__user-name">
+          {name}
+        </span>
+        <span className="property__user-status">
+          {isPro && 'Pro'}
+        </span>
```

Comment:
Наверно имеет смысл сделать так, семантически более верно
```suggestion
        {isPro && <span className="property__user-status">
          'Pro'
        </span>}
```